pkgs/applications/networking/filebrowser/default.nix

```diff
@@ -55,7 +55,7 @@ buildGoModule rec {
     description = "Filebrowser is a web application for managing files and directories";
     homepage = "https://filebrowser.org";
     license = licenses.asl20;
-    maintainers = with maintainers; [ nielsegberts ];
+    maintainers = with maintainers; [ HritwikSinghal ];
```

Comment:
Welcome to the world of nixpkgs pull requests, and great that you want to maintain this package! You need to add yourself to `maintainers/maintainer-list.nix`, else your handle will not resolve :-)

What about the previous maintainer? Do you know that they no longer want to be listed? If not, I'd suggest to just add yourself (in alphabetic order) to them.

I'm not familiar with filebrowser, but this commit looks solid. Thank you for your work!

Reply:
Hi, i have added myself in nix-maintainer list and have also made the required changes. let me know if any more changes are needed. :)